panel/reactive.py

```diff
@@ -11,44 +11,44 @@
 import re
 import sys
 import textwrap
-
 from collections import Counter, defaultdict, namedtuple
 from functools import partial
 from pprint import pformat
-from typing import (
-    TYPE_CHECKING, Any, Callable, Dict, Iterable, List, Mapping,
-    Optional, Set, Tuple, Type, Union
-)
+from typing import (TYPE_CHECKING, Any, Callable, Dict, Iterable, List,
+                    Mapping, Optional, Set, Tuple, Type, Union)
 
 import bleach
 import numpy as np
-import param # type: ignore
-
 from bokeh.model import DataModel
+
+import param  # type: ignore
 from param.parameterized import ParameterizedMetaclass, Watcher
 
 from .io.document import unlocked
 from .io.model import hold
 from .io.notebook import push
 from .io.state import set_curdoc, state
-from .models.reactive_html import (
-    DOMEvent, ReactiveHTML as _BkReactiveHTML, ReactiveHTMLParser
-)
+from .models.reactive_html import DOMEvent
+from .models.reactive_html import ReactiveHTML as _BkReactiveHTML
+from .models.reactive_html import ReactiveHTMLParser
 from .util import edit_readonly, escape, updating
 from .viewable import Layoutable, Renderable, Viewable
 
 if TYPE_CHECKING:
     import pandas as pd
-
     from bokeh.document import Document
     from bokeh.events import Event
     from bokeh.model import Model
     from bokeh.models.sources import DataDict, Patches
+    from holoviews.core.dimension import Dimensioned
     from pyviz_comms import Comm
 
     from .layout.base import Panel
     from .links import Callback, Link
 
+    # Type Alias
+    JSLinkTarget=Union[Viewable, Model, 'Dimensioned']
```

Comment:
I would like to make a Type Alias. Something that can be defined in one place and used where ever that specific type is needed.

I am not sure this is the right way. I believe from Python 3.10 you can/ should type annotate it `TypeAlias`.

```
JSLinkTarget: TypeAlias=Union[Viewable, Model, 'Dimensioned']
```